feature-success-impl/src/main/kotlin/jp/co/soramitsu/success/presentation/SuccessViewModel.kt

```diff
@@ -127,7 +127,4 @@ class SuccessViewModel @Inject constructor(
     }
 }
 
-private fun String.shorten() = when {
-    length < 20 -> this
-    else -> "${take(5)}...${takeLast(5)}"
-}
+private fun String.shorten() = "${take(5)}...${takeLast(5)}"
```

Comment:
Why don't you used shorten() from AddressExt?

Reply:
`shorten()` in `AddressExt` has logic only for addresses. `shorten()` for hash has a little bit different logic (without checking length). I'll rename these functions for increasing code readability.